sailfishusermanager.cpp

```diff
@@ -34,11 +35,20 @@ const int HOME_MODE = 0700;
 const int QUIT_TIMEOUT = 60 * 1000; // One minute quit timeout
 const int MAX_RESERVED_UID = 99999;
 const int OWNER_USER_UID = 100000;
+const auto SYSTEMD_MANAGER_DEST = QStringLiteral("org.freedesktop.systemd1");
```

Comment:
This is service name so maybe SYSTEMD_MANAGER_SERVICE instead.

Reply:
Yep